src/modules/core/components/CSVUploader/CSVUploader.tsx

```diff
@@ -73,11 +80,12 @@ const CSVUploader = ({
         accept: MIME_TYPES,
       }}
       customErrorMessage={error}
-      inputStatusAppearance={{ theme: 'minimal', textSpace: 'wrap' }}
+      inputStatusAppearance={{ theme: 'fat', textSpace: 'wrap' }}
```

Comment:
Are we sure we want change it for all cases?

Reply:
@alicjakujawa This changes only for the 2 cases where addresses for Whitelisting can be provided, inside the Whitelist Extension page, and here in ManageWhitelist modal. Therefore, i wanted the look and feel to be consistent. Besides which, the coloured dot was not always present with 'minimal' theme.